wgpu-types/src/lib.rs

```diff
@@ -3561,6 +3580,7 @@ impl TextureFormat {
             | Self::Rg16Uint
             | Self::Rgba16Uint
             | Self::R32Uint
+            | Self::R64Uint
```

Comment:
We should probably return `None` here if this is only meant to work with atomic ops.

Reply:
regular sampling should work with this format

Reply:
Will it on the APIs we emulate it on?

Reply:
I need it to, so consider that a blocking issue on the PR that actually implements this. Doesnt matter that it doesnt work with this PR because creating this texture errors every time anyways